packages/www/pages/jobs/[slug].tsx

```diff
@@ -140,15 +132,9 @@ const Page = ({
 };
 
 export async function getStaticPaths() {
-  const { allJob } = await request(
-    "https://dp4k3mpw.api.sanity.io/v1/graphql/production/default",
-    print(allJobs),
-    {
-      where: {},
-    }
-  );
+  const allJob = await getJobs();
   let paths = [];
-  allJob.map((page) => paths.push({ params: { slug: page.slug.current } }));
+  allJob.map((page) => paths.push({ params: { slug: page.id } }));
```

Comment:
@FullStackTiger is it possible to use a pretty slug for this? ie `livepeer.com/jobs/protocol-engineer`?

Reply:
If the title contains special symbols like "Marketing Manager / Sr. Marketing Manager", it is complicated a little.